android/src/main/java/com/bitmovin/player/reactnative/RNPlayerView.kt

```diff
@@ -303,3 +324,12 @@ data class RNPlayerViewConfigWrapper(
     val playerViewConfig: PlayerViewConfig?,
     val pictureInPictureConfig: RNPictureInPictureHandler.PictureInPictureConfig?,
 )
+
+data class RNStyleConfigWrapper(
```

Comment:
NIT: not sure if the RN prefix is needed as there is no such wrapper in the Android Player SDK which we could confuse this with.
```suggestion
data class StyleConfigWrapper(
```

Reply:
I would keep it to be aligned with other wrappers (eg a few lines above `RNPlayerViewConfigWrapper`)